pkg/platform/allocator/vacate_test.go

```diff
@@ -599,6 +625,27 @@ func TestCheckVacateFailures(t *testing.T) {
 				errors.New("cluster [123456789][apm] failed vacating, reason: code: unknown, message: an apm error message"),
 			}},
 		},
+		{
+			name: "Returns na apm error on Kibana vacate failure",
```

Comment:
I believe the test name was born by copy- 🍝 :trollface:

Reply:
This test is for CheckVacateFailures, it's not verifying any Cluster IDs

Reply:
haha, this made me laugh so much